doc/source/serve/doc_code/deployment_graph_combine_two_nodes_with_passing_input_parallel.py

```diff
@@ -0,0 +1,31 @@
+import ray
+from ray import serve
+from ray.serve.deployment_graph import InputNode
+
+
+ray.init()
+serve.start()
+
+
+@serve.deployment
+class Model:
+    def __init__(self, weight):
+        self.weight = weight
+
+    def forward(self, input):
+        return input + self.weight
+
+
+@serve.deployment
+def combine(value_refs):
+    return sum(ray.get(value_refs))
```

Comment:
as a follow-up, we should do a simple benchmark on the wide fanout pattern with larger intermediate data size (let's say 50MB) and ensure the added latency makes sense and confirm we're not:

1) Making redundant copies of data 
2) Not leaking memory over time with repeated execution

Reply:
Sg! I will take it as a separate task, not going to follow up in this pr.